Form/Type/ApiMediaType.php

```diff
@@ -85,7 +85,10 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getParent()
     {
-        return 'sonata_media_api_form_doctrine_media';
+        // NEXT_MAJOR: Return 'Sonata\MediaBundle\Form\Type\ApiDoctrineMediaType'
```

Comment:
Please specify on which Symfony version we can remove this hack.

For example:

> Check to be removed when bumping Symfony version to 2.8+